sp800-63a/sec6_dc.md

```diff
@@ -2,64 +2,46 @@
 
 <div class="breaker"></div>
 
-# 6. Leveraging Antecedent Proofing Events
+# <a name="derived-authN"></a> 6. Derived Authenticators
 
 _This section is normative._
 
-Possession of an authenticator, bound to an identity proofed at a given IAL, should allow a claimant to obtain additional authenticators, per CSP policy.  This section details requirements for two specific use cases:
+The expression of IAL and AAL as separate ordinals allows organizations to issue any type or number of authenticators they want regardless of how or if a subject is identity proofed.  This is one of the benefits of the separation - an organization can issue authenticators suitable for use by subscribers regardless of the required IAL to access the service.  However, there are instances when a CSP will want to bind a new authenticator to an existing, identity proofed record, or credential. Since it is beneficial to the subject and CSP to minimize the number of times the identity proofing process needs to be repeated, this can be accomplished by proving possession and successful authentication of an existing authenticator that is already bound to a proofed digital identity. Authenticators issued and bound to existing identity proofed records are considered to be *derived authenticators*.
```

Comment:
You're right, not sure it will make the text any easier, but it is all about the credential (proofing record) not authentication.  Let me see what I can do.